pkgs/tools/networking/gp-saml-gui/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, python3Packages, fetchFromGitHub, gtk3, gobject-introspection, webkitgtk,
+  wrapGAppsHook, glib-networking }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "gp-saml-gui";
+  version = "c557c32ce9429fec2d7054f63195b30b97cba47d";
```

Comment:
Please use unstable-YYYY-MM-DD.

Reply:
Ah, thanks! I wasn't sure how to version this since the author doesn't do versioned releases. Updated.